patent.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import requests
+import json
+
+
+def patent(**kwargs):
+    patent_url = 'https://api.nasa.gov/patents/content'
+    get_params = {}
```

Comment:
That is me not checking my work. I left kwargs in the get_response line to validate it would work and when it did I didn't go back and remove the get_params dict.